src/temperatureConversion.js

```diff
@@ -0,0 +1,43 @@
+let preferedScale = 'celsius';
+
+const scales = {
+  celsius: {
+    conversion(x) {
+      return ((x - 32) * 5) / 9;
+    },
+    symbol: 'C',
+  },
+
+  fahrenheit: {
+    conversion(x) {
+      return (x * 9) / 5 + 32;
+    },
+    symbol: 'F',
+  },
+};
+
+function switchTemperatureScales() {
+  const available = Object.keys(scales);
+  preferedScale = available.filter((scale) => scale !== preferedScale)[0];
+}
+
+function createTemperatureString(number, scale) {
+  return `${Math.round(scale.conversion(number))} °${scale.symbol}`;
+}
+
+function getTemperatureNumbers(elements) {
+  const numbers = elements.map((element) => element.textContent.split(' ')[0]);
```

Comment:
good practice would be to wrap elements to (elements || []). If elements will come undefined or something, it will throw otherwise. And that happens in javascript soo frequently, you couldn't imagine. That's why we always have to check if values exists in places like this